doc/sal_constraints_and_recommendations.rst

```diff
@@ -452,6 +453,15 @@ e.g. for ATTCS in C++:
     ATTCS_shared_SimpleSetA (declared globally)
     ATTCS_someEvent_SpecificSetA (datum specific)
 
+Define it using the following format.
+
+1. In ``CSCName_Events.xml`` create an ``Enumeration`` tag just before the first ``SAlEvent`` tag in the file.
+2. Fill out the Enumeration with comma delimited values using the following naming format ``itemName_value``.
+3. Repeat for as many enumerations as you require.
+4. Optionally, add the enumerations to ``ts_idl`` by creating a ``lsst.ts.idl.enums.{csc_name}`` module.
```

Comment:
Can you remove the "Optionally"?

Reply:
Done.

Reply:
Well they might be cases (C++ only) when this is not needed. And I believe we target to autogenerate those (which shall not be so difficult..it's just XML parsing, I can add this feature if requested to make_idl_files.py)

Reply:
But it is helpful for ts_salobj remotes.